.gitignore

```diff
@@ -7,5 +7,6 @@ coverage.txt
 .idea
 .vscode
 tools-stamp
-.vscode
```

Comment:
[Non Blocking]
Probably this should be kept as it will be created when creating debug configs for examples and we don't want those to be committed to

Reply:
Thanks for flagging!

I intended on adding `.__debug_bin` to this file because VSCode creates it when running tests in debug mode. 

However, removing the `.` prefix from `.vscode` was unintentional. I think my editor removed it because there is already a `.vscode` entry on line 8 of this file. I'll remove both all modifications to this file from this PR b/c the `.__debug_bin` modification can be added separately.

Reply:
resolved in https://github.com/celestiaorg/celestia-app/pull/937/commits/4324bd2909f941dcaab758921833a3263ac9b5b5